src/jit/morph.cpp

```diff
@@ -9445,8 +9463,19 @@ GenTreePtr Compiler::fgMorphOneAsgBlockOp(GenTreePtr tree)
 
             if (!fgIsIndirOfAddrOfLocal(dest))
             {
-                dest->gtFlags |= (GTF_EXCEPT | GTF_GLOB_REF | GTF_IND_TGTANYWHERE);
-                tree->gtFlags |= (GTF_EXCEPT | GTF_GLOB_REF | GTF_IND_TGTANYWHERE);
+                dest->gtFlags |= (GTF_GLOB_REF | GTF_IND_TGTANYWHERE);
+                tree->gtFlags |= (GTF_GLOB_REF | GTF_IND_TGTANYWHERE);
```

Comment:
Is it correct to set  `GTF_IND_TGTANYWHERE` on `tree`? Seems like it is a `GT_ASG`, not a `GT_IND`.

Reply:
It's pre-existing but I don't think it's correct. I'll try to remove it.